byway/src/main/webapp/style.css

```diff
@@ -7,3 +7,153 @@
 #greeting-container {
   margin-top: 20px;
 }
+
```

Comment:
Fix some spacing across css file. Also, not using greeting container and other stuff at the start of the file, so we can remove

Reply:
done!